doc/conf/Doxyfile.in

```diff
@@ -7,8 +7,8 @@ PROJECT_BRIEF          = "Ginkgo is a numerical linear algebra library targeting
 INLINE_INHERITED_MEMB  = NO
```

Comment:
It's actually for what is up there, CMake variables can be used for populating the `PROJECT_NAME` and `PROJECT_BRIEF`. I think the respective variables would be `PROJECT_NAME` and  `PROJECT_DESCRIPTION`. Do not hesitate to update the description in the main CMakeLists.txt if you want to improve it at the same time.

In addition for the `PROJECT_NUMBER`, in `cmake/information_helpers.cmake` there is a `ginkgo_git_information` macro which provides the variable `GINKGO_GIT_BRANCH` with the branch name. Of course it relies on having Ginkgo with a git repository, but there is maybe a way to integrate this in the documentation to indicate the actual branch for which this documentation was generated.